sql/core/src/test/scala/org/apache/spark/sql/DataFrameFunctionsSuite.scala

```diff
@@ -331,6 +331,15 @@ class DataFrameFunctionsSuite extends QueryTest with SharedSparkSession {
     checkAnswer(df.select(nullif(lit(5), lit(5))), Seq(Row(null)))
   }
 
+  test("nullifzero function") {
+    val df = Seq((0)).toDF("a")
+    checkAnswer(df.selectExpr("nullifzero(0)"), Seq(Row(null)))
```

Comment:
You have already tested the SQL function in the previous PR. I do believe this check is not necessary.

Reply:
sounds good, removed.